WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -318,7 +318,17 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
         guard let toolBar = navigationController?.toolbar else {
             return
         }
-        toolBar.barTintColor = UIColor(light: .white, dark: .appBarBackground)
+
+        if #available(iOS 15.0, *) {
+            let appearance = UIToolbarAppearance()
+            appearance.configureWithDefaultBackground()
+            appearance.backgroundColor = UIColor(light: .white, dark: .appBarBackground)
```

Comment:
As far as I can tell there's no "tint color" concept so while the background of the toolbar will match iOS 14, it loses the slight blur + transparency effect.